management/server/activity/sqlite/sqlite.go

```diff
@@ -140,6 +188,22 @@ func (store *Store) Save(event *activity.Event) (*activity.Event, error) {
 	return eventCopy, nil
 }
 
+func (store *Store) SaveWithDeletedUserEmail(event *activity.Event, email string) (*activity.Event, error) {
+	email = store.emailEncrypt.Encrypt(email)
+
+	stmt, err := store.db.Prepare(insertDeleteUserStatement)
```

Comment:
I suggest not preparing a statement each time we call this method. Let's do it once in `NewSQLiteStore`. Honestly, I would expect golangci-lint to complain about it. Could it be that the PR needs a rebase on the main branch?

Reply:
I will fix it

Reply:
My concern here is that every time `SaveWithDeletedUserEmail` is being called we recreate a statement by  `store.db.Prepare`. I can't explain it any better than in this document: https://go.dev/doc/database/prepared-statements.
```
When you expect to execute the same SQL repeatedly, you can use an sql.Stmt to prepare the SQL statement in advance, then execute it as needed.
```

See how the rest of the statements are prepared in `NewSQLiteStore`: https://github.com/netbirdio/netbird/blob/bb791d59f3cf55437d2e7749f9df2759ae019b80/management/server/activity/sqlite/sqlite.go#L57